Library/OcAppleKernelLib/CommonPatches.c

```diff
@@ -60,12 +60,20 @@ PatchAppleCpuPmCfgLock (
   UINT8  *WalkerEnd;
   UINT8  *WalkerTmp;
 
+  //
+  // NOTE: As of macOS 13.0 AICPUPM kext is removed.
+  // However, we may remove this check later, if an older version can be injected correctly
+  // such that it will patched.
+  //
   if (OcMatchDarwinVersion (KernelVersion, KERNEL_VERSION_VENTURA_MIN, 0)) {
     DEBUG ((DEBUG_INFO, "OCAK: Skipping AppleCpuPmCfgLock patch on %u\n", KernelVersion));
     return EFI_SUCCESS;
   }
 
-  ASSERT (Patcher != NULL);
+  if (Patcher == NULL) {
+    DEBUG ((DEBUG_INFO, "OCAK: Skipping %a on NULL Patcher on %u\n", __func__, KernelVersion));
+    return EFI_SUCCESS;
```

Comment:
Why is this EFI_SUCCESS? If the kext is missing and we did not expect it, I would assume it is an error. Should be EFI_NOT_FOUND. Otherwise looks good. Please merge.

Reply:
https://github.com/acidanthera/OpenCorePkg/pull/359/commits/3e35e5d8c8445c819fd7af4bf8949d444d2ddcdd